components/json_page/index.coffee

```diff
@@ -30,13 +30,13 @@ module.exports = class JSONPage
         .end (err, res) =>
```

Comment:
Wait, confused, we are catching the error in the right order here. Am I missing something?